### PR TITLE
Denormalization layer

### PR DESCRIPTION
**Pull request by markub3327**
@gadagashwini

Solved #16284

### CODE REVIEW

**Comment by fchollet** [comment]
Thanks for the PR. I see we have an `invert` argument in the preprocessing layers [`StringLookup`](https://keras.io/api/layers/preprocessing_layers/categorical/string_lookup) and
[`IntegerLookup`](https://keras.io/api/layers/preprocessing_layers/categorical/integer_lookup). Perhaps this use case could be covered in a simpler fashion by also adding `invert` to the `Normalization` layer?

@mattdangerw what do you think?

**Comment by mattdangerw** [comment]
Yeah, I think `invert` as an argument make sense. Could you add a unit test seeing if this would also be workable for an adapted layer? I think the following snippet should work, but it would be good to check.

```
norm = keras.layers.Normalization(axis=None)
norm.adapt(some_data)
inv_norm = keras.layers.Normalization(
    mean=norm.mean,
    variance=norm.variance,
    axis=norm.axis,
    invert=True)
```

We should probably also show an example of this in the docstring, taking some data, first normalizing, then inverting. For both the adapted and non-adapted cases.

**Comment by markub3327** [comment]
@mattdangerw

I create both unit tests.

**Comment by mattdangerw** [comment]
@markub3327 thanks. Could you add an example of `invert` in the docstring as well?

**Comment by markub3327** [comment]
Yes, sorry I forgotten. It's done.

**Review by fchollet — APPROVED** [review]
LGTM, thanks. I'll do a round of copyedits in post-processing.

**Comment by markub3327** [comment]
@fchollet 

Ok. Thanks.
